lib/cqm/converter/qdm_patient.rb

```diff
@@ -288,7 +288,7 @@ def unpack_patient_characteristics(patient, record)
       # Convert patient characteristic race.
       race = get_data_elements(patient, 'patient_characteristic', 'race').first
       race_code = race.dataElementCodes.first.symbolize_keys if race.dataElementCodes.any?
-      record.race = { code: race_code[:code], name: race_code[:descriptor], codeSystem: race_code[:codeSystem] } if race_code
+      record.race = { code: race_code[:code], name: race_code[:descriptor], codeSystem: 'CDC Race' } if race_code
```

Comment:
I assume we don't need to do anything in this file about the gender code, because the code coming in should already be `AdministrativeGender` and not `AdministrativeSex`?